pyangbind/lib/serialise.py

```diff
@@ -407,7 +407,7 @@ def generate_ietf_tree(obj, parent_namespace=None, flt=False, with_defaults=None
             # were a scalar.
             return obj
 
-        d = {}
+        d = OrderedDict()
```

Comment:
`obj._pyangbind_elements` doesn't have a strict requirement for ordering today (it will get defined based on the ordering in the schema). However, I think the reason to implement this is to have keys of lists ordered first -- which isn't guaranteed. For example:

```yang
module bar {
	prefix "b";
	namespace "urn:b";

	list foo {
		key "baz bat";
	
		leaf bat { type string; }
		leaf baz { type string; }
	}
}
```

Results in the output:

```
▶ grep "pyangbind_elements" binding.py | grep bat
  _pyangbind_elements = OrderedDict([('bat', bat), ('baz', baz), ])
```

But I think that you want 'baz' to be output first, since RFC6020 says:

```
   The list's key nodes are encoded as subelements to the list's
   identifier element, in the same order as they are defined within the
   "key" statement.
```

So, some pre-processing is likely required here AFAICS.